generators/aws-containers/README.md

```diff
@@ -0,0 +1,24 @@
+# AWS JHipster Generator - _A new hope_
+
+## AWS
+### ECR
+#### Authentication Token
+The token received by ECR to authenticate is structured as follow. We'll only take the first object.
+```json
+  {  
+     "authorizationToken":"AUTHORIZATION_TOKEN",
+     "expiresAt":"2017-12-12T10:52:24.810Z",
+     "proxyEndpoint":"https://foo.ecr.region.amazonaws.com"
+  }
+```
+
+## Development
+### Dependencies
+
+```
+# NPM
+npm i --save-dev aws-sdk@^2.167.0 progress@2.0.0 ora@1.3.0
```

Comment:
Can the versions be fixed for both yarn and npm?

Reply:
I'm not sure what you mean here. The installation instructions for both yarn and npm is the same

Reply:
can we put the version numbers in a constant and use that in both places so that at least you only need to change it once

Reply:
It is not possible to use variables in Markdown files but we could put constants in `aws-client.js` and maybe put a TODO comment to remind to update the doc

Reply:
By fixed I meant without `^` in the versions, like `npm i --save-dev aws-sdk@2.167.0 progress@2.0.0 ora@1.3.0`

Reply:
Oh sorry didn't notice that this was the readme of the sub gen itself